dataikuapi/fm/instances.py

```diff
@@ -175,7 +177,7 @@ def create(self):
 class FMInstance(object):
     """
     A handle to interact with a DSS instance.
-    Do not create this directly, use :meth:`FMClient.get_instance` or :meth: `FMClient.new_instance_creator`
+    Do not create this directly, use :meth:`FMClient.get_instance` or :meth:`FMClient.new_instance_creator`
```

Comment:
These method links don't seem to work, maybe you need to do something like `dataikuapi.FMClient.get_instance`?

Reply:
need prefix indeed, and new_instance is only available on specific cloud classes